crates/solver/src/settlement_submission/submitter/eden_api.rs

```diff
@@ -51,20 +54,24 @@ impl EdenApi {
         };
         let params =
             serde_json::to_value(Bytes(raw_signed_transaction)).context("failed to serialize")?;
+        let request = helpers::build_request(1, "eth_sendSlotTx", vec![params]);
 
         let response = self
-            .rpc
-            .transport()
-            .execute("eth_sendSlotTx", vec![params])
+            .client
+            .post(self.url.clone())
+            .json(&request)
+            .send()
             .await
-            .context("transport failed")?;
-        tracing::debug!("response from eden: {:?}", response);
+            .context("failed sending request")?
+            .text()
+            .await
+            .context("failed converting to text")?;
+        tracing::debug!("response from eden: {}", response);
```

Comment:
ubernit: `tracing` has a cool macro feature for you to add values to the log message that makes it slightly more concise (also I think it allows tighter integration with subscribers, but I'm not 100% sure about that):
```suggestion
        tracing::debug!(%response, "response from eden");
```

Reply:
Cool, updated.